7Cav-Alive.Altis/init/fn_serverPreInit.sqf

```diff
@@ -1,3 +1,28 @@
 if (not isServer && hasInterface) exitWith {};
 
+#include "\serverscripts\zeusserverscripts\secretKey.sqf" //Includes the zeus security key.sqf so we can reference it for zeus IDs
+#include "..\scripts\baseCleanup\clearItemsBASE_switch.sqf" //Base script
+#include "..\scripts\server\deleteWeaponHolder.sqf" //Includes the deleteWeaponHolder script so everything can reference it 
+
+missionNamespace setVariable["LOYALTY_CIPHER", CIPHERSECRETKEY, true]; //variable init for loyalty cipher
+missionNamespace setVariable["endMissionNumber",0]; //Variable init for missionEnd scripts located in scripts/missionEnd
+
+//Initial Task Creation for all strongpoints
+//Airfield
+private _endMission = [BLUFOR, "Deactivate the Airfield!" Term1, ["Deactivate the satellite to prevent the enemy air commander from being able to call in air strikes!" _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
+["Deactivate the Airfield!", "Attack"] call BIS_fnc_taskSetType;
+
+//Armor
+private _endMission = [BLUFOR, "Deactivate the Armored Corps!" Term2, ["Deactivate the satellite to prevent the enemy armored commander from being able to call in armored support!" _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
+["Deactivate the Armored Corps", "Attack"] call BIS_fnc_taskSetType;
+
+//Mechanized
+private _endMission = [BLUFOR, "Deactivate the Mechanized Corps!" Term3, ["Deactivate the satellite to prevent the enemy mechanized infantry commander from being able to call in mechanized infantry support!" _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
+["Deactivate the Mechanized Corps!",  "Attack"] call BIS_fnc_taskSetType;
+
+//Special Operations
+private _endMission = [BLUFOR, "Deactivate the Specops headquarters!" Term4, ["Deactivate the satellite to prevent the enemy special operations commander from being able to sabotage our efforts!" _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
+["Deactivate the Specops headquarters!", "Attack"] call BIS_fnc_taskSetType;
+
 [] execVM "scripts\server\curator.sqf";
+[] execVM "scripts\client\cleanUpLoop.sqf"; 
```

Comment:
Base cleanup should be a server script, not a client script. If its a client script, then every client will be running the clean up which puts unneeded load on both the clients and on the server as it's having to clean very often (if 5 people enter a minute apart, the script would run once every minute).

In fact, your script is already running on the server because you run `([getPos headquarters] + _headquartersArea) remoteExec ["SERVER_DeleteWeaponHolders", 2]`

So get rid of `remoteExec` part of the whole thing, and just have the loop running on the server instead of the clients. Also, just put all the functions into the same file. 

So have a file `scripts\server\baseCleanup.sqf`

Which contains:
```sqf
SERVER_BaseCleanup = {
	private _headquartersArea = triggerArea headquarters;
	{ deleteVehicle _x } forEach (allDeadMen inAreaArray headquarters);
	([getPos headquarters] + _headquartersArea) call SERVER_DeleteWeaponHolders;
};

SERVER_DeleteWeaponHolders =
{
    params ["_center", "_a", "_b", "_angle", "_isRectangle", "_c"];

    {
        deleteVehicle _x;
    } forEach (nearestObjects [_center, ["WeaponHolder"], sqrt (_a*_a + _b*_b)]) inAreaArray [_center, _a, _b, _angle, _isRectangle, _c];
}; 

[] spawn {
  while {true} do {
      [] call SERVER_BaseCleanup; 
      sleep 300;
    };
}; 
```

Then, you can `execVM` the file you've just made, just like before.